app/routes/irrigated-land.js

```diff
@@ -40,7 +40,7 @@ function createModel (irrigatedLandCurrent, irrigatedLandTarget, errorMessageLis
       id: 'irrigatedLandTarget',
       name: 'irrigatedLandTarget',
       hint: {
-        text: 'Enter figure in hectares'
+        html: '<font color="black"> Hectares </font><br/> Enter figure in hectares, for example 543.5'
```

Comment:
`<span class="govuk-label govuk-!-font-weight-bold"> Hectares </span>`

Reply:
I think this is better than `<font color = "black">` - we should always use govuk stylesheets, however `govuk-!-font-weight-bold` may or may not be want the content design wants - please check it out locally & decide

Reply:
done